### PR TITLE
Remove iframe and use div[contenteditable] instead

### PR DESCRIPTION
**Pull request by besuhoff**

### CODE REVIEW

File: src/javascript/app/ng-wig/ngWigDirective.js [hunk 1]
```diff
@@ -3,8 +3,7 @@ angular.module('ngWig').directive('ngWig', function () {
       return {
         scope: {
           content: '=ngWig',
-          debug: '&',
-          cssPath: '@'
+          debug: '&'
```
Comment (stevermeister) [inline src/javascript/app/ng-wig/ngWigDirective.js]: do we still use it?

File: src/javascript/app/ng-wig/ngWigEditableDirective.js [hunk 2]
```diff
@@ -1,43 +1,43 @@
 angular.module('ngWig').directive('ngWigEditable', function () {
       function init(scope, $element, attrs, ctrl) {
-        var $document = $element[0].contentDocument,
-            $body;
-        $document.open();
-        $document.write('<!DOCTYPE html><html style="min-height: 100%"><head>'+ (scope.cssPath ? ('<link href="'+ scope.cssPath +'" rel="stylesheet" type="text/css">') : '') + '</head><body contenteditable="true" style="height:100%; margin: 0; padding: 8px;box-sizing: border-box;"></body></html>');
-        $document.close();
+        var document = $element[0].ownerDocument;
```
Comment (stevermeister) [inline src/javascript/app/ng-wig/ngWigEditableDirective.js]: why not just $window.document?

File: src/javascript/app/ng-wig/ngWigEditableDirective.js [hunk 3]
```diff
@@ -1,43 +1,43 @@
...
         ctrl.$render = function () {
-          $body[0].innerHTML = ctrl.$viewValue || '';
+          $element.html(ctrl.$viewValue || '');
         };
 
         //view --> model
-        $body.bind('blur keyup change paste', function () {
+        $element.bind('blur keyup change paste', function () {
           resizeEditor();
           scope.$apply(function blurkeyup() {
```
Comment (stevermeister) [inline src/javascript/app/ng-wig/ngWigEditableDirective.js]: do we need apply in this case?

File: src/javascript/app/ng-wig/ngWigEditableDirective.js [hunk 4]
```diff
@@ -1,43 +1,43 @@
...
-            ctrl.$setViewValue($body.html());
+            ctrl.$setViewValue($element.html());
           });
         });
 
         scope.$on('execCommand', function (event, params) {
-          var sel = $document.selection,
+          $element[0].focus();
+
+            var sel = document.selection,
```
Comment (stevermeister) [inline src/javascript/app/ng-wig/ngWigEditableDirective.js]: sel - > selection/textSelection

File: src/javascript/app/ng-wig/ngWigEditableDirective.js (hunk 2 above)
Comment (besuhoff) [inline src/javascript/app/ng-wig/ngWigEditableDirective.js]: because what if it's within iframe or something?

**Comment by besuhoff** [comment]
@stevermeister updated lol

**Pull request merged**